README.md

```diff
@@ -436,6 +436,10 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct][]. For more information see the
 [Code of Conduct FAQ][] or contact [opencode@microsoft.com][] with any additional questions or comments.
 
+# Encouraging Contributors to Code Review
+
+It can sometimes take time for maintainers to review a PR, and if a PR goes through several iterations of maintainer review, the lifetime of the PR is extended even further. If contributors are willing to review PRs, it can oftentimes accelerate the review process as it allows the PR to be in a more finished state before maintainers review, and fewer maintainer review iterations are needed. It has always been appreciated when contributors are willing to review each others' PRs, but this has not been explicitly said anywhere.

```

Comment:
Instead of PR , you must write the  Pull Request .  Also  there is an **'** before others , request you to omit it

Reply:
Also The Code Format Text is failing as you have written everything in one line. What you can do is just refer from the readme.md file  on how they wrote documentation(if you see the have added a new line after some words)

Reply:
This is just a copy and paste of the issue description.

Reply:
I think we should add something to the wiki (probably to the Contribution Guidelines page that's on my plate) but probably not the README, since the README is generally for someone's first introduction to the repo, but the advice to review PRs should be directed at contributors who are increasingly familiar with the codebase.